gcloud/storage/test_bucket.py

```diff
@@ -1000,10 +1000,8 @@ def __init__(self, *responses):
 
     @staticmethod
     def _is_bucket_path(path):
-        if not path.startswith('/b/'):  # pragma: NO COVER
-            return False
         # Now just ensure the path only has /b/ and one more segment.
-        return path.count('/') == 2
+        return path.startswith('/b/') and path.count('/') == 2
```

Comment:
I'm surprised that this one doesn't trip on branch coverage (or was the `# pragma: NO COVER` just a decoy?)